src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,49 @@
+package duke;
```

Comment:
I like how the project is organised; every class is in a package, and the Tasks are in their own subpackage as well.